test/integration/test/no_travis/pagekite_access_test.js

```diff
@@ -1,96 +0,0 @@
-'use strict';
```

Comment:
It seems these tests are not present somewhere else in the PR. Should the file be renamed?